config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -1160,3 +1160,44 @@ presubmits:
           requests:
             cpu: 4
             memory: 6Gi
+  - name: pull-kubernetes-e2e-containerd-gce-serial
```

Comment:
Please rename this to include something about disruptive tests. We already have a serial job and I think this will confuse people.